lib/data/network/media/media_api.dart

```diff
@@ -34,4 +35,23 @@ class UploadFileAPI {
 
     return UploadFileResponse.fromJson(response);
   }
+
+  Future<UrlPreviewResponse> getUrlPreview({
+    required Uri uri,
+    int? preferredPreviewTime,
+  }) async {
+    final response = await _client
+        .get(
+          HomeserverEndpoint.getPreviewUrlServicePath
+              .generateHomeserverEndpoint(),
+          queryParameters: {
+            'url': uri.toString(),
+            if (preferredPreviewTime != null) 'ts': preferredPreviewTime,
```

Comment:
shouldn't it be ? 
```suggestion
            preferredPreviewTime ?? 'ts',
```

Reply:
<img width="743" alt="Screenshot 2023-09-18 at 23 35 19" src="https://github.com/linagora/twake-on-matrix/assets/99852347/6d2d5889-86a4-42db-87e3-d998faf5647b">

`ts` is a parameter that can't use `preferredPreviewTime ?? 'ts'`.